.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+﻿# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
+# Delete branch after closing a stale PR
+delete-branch: true
```

Comment:
I didn't find this rule in the documentation, can you point me to the appropriate article?

Reply:
https://github.com/actions/stale#:~:text=delete%2Dbranch,a%20stale%20PR

Reply:
If you visit this link: https://github.com/marketplace/stale
and visit their github repo: https://github.com/probot/stale
you can see that they have another repository

Reply:
oh, I found a link that pointed me to that repo.
Anyways, so we are limited to pass only a few configuration params.